workspaces/web/cypress/integration/listening.feature

```diff
@@ -59,11 +60,21 @@ Feature: Listening exercise
     And I click "Submit"
     Then I don't see a "Skip" button
     And I click "Continue"
-    Then I see a panel with only a skip and a finish early button
+    Then I see a "Skip" button
+    And I see a "Can't listen now" button
+    And I see a "Finish early" button
 
-  Scenario: Going to the next challenge with keyboard only
+  Scenario: Skipping a listening excercise
```

Comment:
Why has this one been renamed?

Reply:
I must've changed it by accident, fixed it with the newest commit

Reply:
amazing, thank you!